doc/iterations/YYYY-MM/2023.6/shapes/shape Migration flow.adoc

```diff
@@ -0,0 +1,27 @@
+= (appetite) Shape
+
+== Problem
+
+In order to show that the view DSL can support everything the compatibility layer support, we should migrate flow to the view DSL. Using a dedicated package named flow, we should add a new project sirius-components-flow-view which should contain the view model for the various flow diagrams.
+
+This view model should be generated using the soon to be introduced Java APIs to create view models. https://github.com/eclipse-sirius/sirius-components/issues/1919
+
+== Key Result
+
+The flow odesign model should be converted to a view model. All concepts used in the flow model should have a respective view concept.
+
+== Solution
+
+Use the the soon to be introduced Java APIs to create view models. https://github.com/eclipse-sirius/sirius-components/issues/1919
+
+=== Scenario
+
+Giving a flow model defined in a odesign file, we should be able to generate a view model witch match all concepts in the odesign model.
```

Comment:
witch -> which
match all concepts -> match all the concepts existing in the odesign flow model